sdk/sqlmanagement/Azure.ResourceManager.Sql/tests/Azure.ResourceManager.Sql.Tests.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TestHelperProjects>Resources201705;Network202004;Storage201906;</TestHelperProjects>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="ServerCrudScenarioTests.cs" />
+  </ItemGroup>  <ItemGroup>
+    <ProjectReference Include="..\src\Azure.ResourceManager.Sql.csproj" />
+  </ItemGroup>  <ItemGroup>
+  <Folder Include="Utilities\" />
+</ItemGroup>  <ItemGroup>
+  <None Include="ServerCrudScenarioTests.cs" />
```

Comment:
Why include this CS file, if it is not compiling, instead, we should remove this from this PR, and add it when we are ready to add these tests.